mlrun/projects/project.py

```diff
@@ -46,6 +46,7 @@
 from ..features import Feature
 from ..model import EntrypointParam, ModelObj
 from ..run import code_to_function, get_object, import_function, new_function
+from ..runtimes.pod import AutoMountType
```

Comment:
-_-, we have multiple runtimes imports in this file, some are relative and some are from, please align